From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Wed, 20 Nov 2024 15:56:29 -0600
Subject: [PATCH] Solus: Don't hide classic theme

Classic theme is required for freecad to use the system Qt theme by default

---
 src/Mod/Start/Gui/ThemeSelectorWidget.cpp | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/Mod/Start/Gui/ThemeSelectorWidget.cpp b/src/Mod/Start/Gui/ThemeSelectorWidget.cpp
index 83a9043caf..5abf9d3970 100644
--- a/src/Mod/Start/Gui/ThemeSelectorWidget.cpp
+++ b/src/Mod/Start/Gui/ThemeSelectorWidget.cpp
@@ -79,14 +79,7 @@ static bool isSystemInDarkMode()
 
 static bool shouldHideClassicTheme()
 {
-    // Classic on macOS and windows 11 with qt6(.4+?) doesn't work when system
-    // is in dark mode and to make matter worse, on macOS there's a setting that
-    // changes mode depending on time of day.
-#if QT_VERSION >= QT_VERSION_CHECK(6, 4, 0) || defined(FC_OS_MACOSX) || defined(FC_OS_WIN32)
-    return true;
-#else
     return false;
-#endif
 }
 
 
